packages/aft/lib/src/commands/generate/generate_workflows_command.dart

```diff
@@ -24,12 +24,27 @@ class GenerateWorkflowsCommand extends AmplifyCommand {
           !falsePositiveExamples.contains(package.name)) {
         continue;
       }
-      // Some packages contain only native code/dependencies and do not need
-      // to be tested by Dart/Flutter analyze or test.
+      // Some packages do not need to be tested.
       final libDir = Directory(p.join(package.path, 'lib'));
       if (!libDir.existsSync()) {
         continue;
       }
+
+      final internalAndroidTestDir =
+          Directory(p.join(package.path, 'android', 'src', 'test'));
+      final externalAndroidPackageTestDir = Directory(
```

Comment:
`internal`/`external` is a bit confusing to me, since I would think of the `_android` package as being "internal".

Maybe we can borrow the language of [federated plugins](https://docs.flutter.dev/development/packages-and-plugins/developing-packages) and use `appFacing`/`platform` instead